class/trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    trimmed = @nameable.correct_name[0, 9]
+    @nameable.correct_name.length >= 10 ? trimmed : @nameable.correct_name
+  end
```

Comment:
- [ ] Great job for implementing the `TrimmerDecorator` 👍👍 However, please make sure to return maximum of 10 characters not 9 as per [this requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Implement%20a%20method%20correct_name%20that%20makes%20sure%20that%20the%20output%20of%20%40nameable.correct_name%20has%20a%20maximum%20of%2010%20characters.%20If%20it%27s%20longer%20it%20should%20trim%20the%20word.). To fix this, you can use the following 👇✨

```ruby
 trimmed = @nameable.correct_name[0, 10]
 # or
 trimmed = @nameable.correct_name[0..9]
```

**_current implementation_**

![image](https://github.com/fickryiman/school-library-oop/assets/106119105/9b1fcafc-ad17-4e06-9197-6060b082a992)

Reply:
Hi @KanzaTahreem,

Fixes the the trimmed name to return 10 characters..